.github/workflows/gradle.yml

```diff
@@ -70,185 +76,330 @@ jobs:
           git config --global user.name "Doordeck Development"
           git config --global user.email "development@doordeck.com"
 
+      - name: Tag for release (if on main)
+        if: github.ref == 'refs/heads/main'
+        run: ./gradlew final -x test
+
+      - name: Store tag
+        id: pre-release-tag
+        run: echo "tag=$(git describe --tags --abbrev=0 | sed 's/^v//')" >> "$GITHUB_OUTPUT" # i.e: 0.74.0
```

Comment:
I *think* you can remove the `sed` part since you seem to be manually re-adding the `v` prefix everywhere you use the tag, so just switch to expecting the full tag name `v0.74.0` and remove the `v` usages elsewhere, but either way is good so I'll approve.

Reply:
That is weird, but I don't see how I can remove the `v` if I don't need it without creating a new variable. For example, in the NuGet publish command, it seems much easier to just append it where needed 👀